src/typed-method-types/chat.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseResponse } from "../types.ts";
+
+type ChatPostMessageOptionalArgs = {
```

Comment:
Can we add JSDoc descriptions here, maybe just lifted [straight from our API docs](https://api.slack.com/methods/chat.postMessage#args)? I think it would be super rad. For example, while `text` is technically optional, i[t is used as a fallback for areas where rich media won't work (like desktop or mobile notifications) as well as in use by screen readers](https://api.slack.com/methods/chat.postMessage#blocks_and_attachments). Putting that info into a `@description` JSDoc would be badass IMO.

Reply:
✅